devtools/maven/src/main/java/io/quarkus/maven/BuildMojo.java

```diff
@@ -116,7 +116,8 @@
     @Parameter(property = "ignoredEntries")
     private String[] ignoredEntries;
 
-    @Parameter(defaultValue = "false")
+    /** Skip the execution of this mojo */
+    @Parameter(defaultValue = "false", property = "quarkus.skip")
```

Comment:
Perhaps we should name it `quarkus.build.skip`? Other mojos might follow the same convention for the skip flag too :wink:

Reply:
+1